src/App.js

```diff
@@ -1,25 +1,34 @@
-import logo from './logo.svg';
+
 import './App.css';
+import Form from "./components/Form.js";
+
+
+
```

Comment:
I think is a good practice to remove the commented code before submit the project